### ISSUE TITLE
Allow always open in default container

### ISSUE THREAD

**Issue by t3pfaffe** [mozilla/multi-account-containers#1874]
[Feature Request]

I would like for it be an option to auto convert tabs to de-contained with certain sites just like you would if that site was set to another container by default. 

Basically my problem is that I have a shopping container. However if I select the name of a product and then use the right click to "Search DuckDuckGo for <highleted text>" and then say click on a YouTube video; that YouTube tab is still in the "shopping" container. A container in which I am not logged in with any google or YouTube accounts. This can actually be desirable in the case that you dont want your google account connected to all the things you are researching before buying. However, for me it can be rather inconvenient with the reduced features in youtube when not logged in.  

**TLDR: I would like an "Always open this site in > Decontained" option in the extension.**

**Comment by Dr-Ari-Gami** [comment]
Same here, I've been wanting this for a long time.

**Comment by maxtar** [comment]
Maybe better always open site in "Not in container" if it hasn't setting "Open in container'?

**Comment by grahamperrin** [comment]
> always open in default container

▶ 2017 issue #503

@t3pfaffe please close this duplicate issue. Thank you.

**Comment by dannycolin** [comment]
Duplicate of #503
